papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,553 @@
+Though the signac package has methods like ``Project.groupby``, which can generate subsets of the project that are grouped by a state point key, there has been no similar feature in signac-flow to allow operations to act on multiple jobs.
+The concept of _aggregation_ provides a straightforward way for users to write and submit operations that act on arbitrary subsets of jobs in a signac data space.
+Just as the groups feature acts as an abstraction over operations, aggregation can be viewed as an abstraction over jobs.
+Operations now can accept multiple job instances as positional arguments though Python's argument unpacking while maintaining backwards compatibility.
+Decorators are used to define aggregation behavior, encompassed in the ``aggregator`` decorator for single operations and in the argument ``aggregator_function`` to ``FlowProject.make_group`` for groups of operations.
+
+.. code-block:: python
+
+    from flow import FlowProject
+
+    @aggregator
+    @FlowProject.operation
+    def operation_on_all_jobs(*jobs):
+        import matplotlib.pyplot as plt
+        import numpy as np
+
+        x = np.array(
+            [job.sp.temperature for job in jobs])
+        y = np.array(
+            [job.doc.activity for job in jobs])
+        fig, ax = plt.subplots()
+        ax.plot(x, y)
+        ax.set_title(
+            "Enzymatic Activity Across Temperature")
+        fig.savefig("enzyme-activity.png")
+
+Like groups, there are many reasons why a user might wish to use aggregation.
+For example, a signac data space that describes weather data for multiple cities in multiple years might want to plot or analyze data that uses ``aggregator.groupby("city")`` to show changes over time for each city in the data space.
+Similarly, aggregating over replicas facilitates computing averaged quantities and errors.
+Another example is submitting aggregates with a fixed number of jobs in each aggregate to enable massive parallelization by breaking a large MPI communicator into a smaller communicator for each independent job, which is necessary for efficient utilization of leadership-class supercomputers like OLCF Summit.
+
+A third concept while not new warrants attention in this section, bundling.
+Bundling is a third way to combine pieces of signac-flow's workflow.
+Whereas aggregates are concerned with jobs and groups operations, bundling is concerned with combining what are effectively units of execution into a single submission script.
+The simplest case of a bundle is a submission script with the same operation being executed for multiple jobs either in parallel or serial.
+Bundling is what allows the script to contain multiple jobs executing the same operation.
+By storing information on individual bundles, signac-flow prevents accidental resubmission just as in the unbundled case.
+While the example above does not use either groups or aggregation, bundles works seamlessly with both.
```

Comment:
We probably need to mention that unlike groups and aggregation, bundling is not a part of the workflow definition -- it's a choice made by the user at submission time.

Reply:
See the changes above and at 298-300 (beginning of the section) to see if you think this resolves this.